cypress/e2e/components/filings/addStaffFiling.cy.ts

```diff
@@ -142,27 +145,31 @@ context('Add Staff Filing', () => {
         response).as('businessComments')
     })
     cy.visitBusinessDashFor('businessInfo/ben/active.json', undefined, false, false, undefined, allFilings, true)
-    cy.get('[data-cy="add-staff-filing"]').should('exist').click()
+    cy.get('[data-cy="add-staff-filing"]').should('exist')
+    cy.get('[data-cy="add-staff-filing"]').click()
 
```

Comment:
In some places, granted, this one might not.
"should" should be safe to chain on, but for example chaining click().get etc ... is not safe. Though it could work, depending on how dom behaves at that moment. 

I just tried to clean it up for reading comprehension a bit, so I could understand better whats happening. 

What is your preferred way of reading it, do you find it more easily to read like this, or when the commands are chained one after the other ?